foodloop-db-files/seeders/03_insertFoodlist.sql

```diff
@@ -1,7 +1,3 @@
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('burger', '12:00:00', 'Monday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('pizza', '12:30:00', 'Tuesday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('drinks', '18:00:00', 'Wednesday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('salad', '13:00:00', 'Monday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('pastry', '12:30:00', 'Tuesday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('rice', '20:00:00', 'Thursday', 1);
-INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('exotic', '12:00:00', 'Friday', 1);
\ No newline at end of file
+INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('chicken rice', '12:00:00', 'Monday', 1);
+INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('pasta', '12:30:00', 'Tuesday', 1);
+INSERT INTO foodloop.foodlist (foodlistName, foodlistTime, foodlistDay, foodlistCurrIdx) VALUES ('biryani', '18:00:00', 'Wednesday', 1);
```

Comment:
please change time values to exclude seconds i.e. `"12:00:00"` -> `"12:00"`. In another PR i have raise i changed the schema to `string` instead of `time`.